reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -129,12 +129,11 @@ def launch_refarch_env(region=None,
     bastion_sg = click.prompt('Specify the the Bastion Security group(example: sg-4afdd24)')
   
 
-  # If the user already provided values, don't bother asking again
-  if rhsm_user is None:
+  if deployment_type in 'openshift-enterprise' and rhsm_user is None:
     rhsm_user = click.prompt("RHSM username?")
-  if rhsm_password is None:
+  if deployment_type in 'openshift-enterprise' and rhsm_password is None:
     rhsm_password = click.prompt("RHSM password?")
-  if rhsm_pool is None:
+  if deployment_type in 'openshift-enterprise' and rhsm_pool is None:
```

Comment:
similar to above, should be '==' instead of 'in' or should be using 'in' with a list.